web-component/styles.css

```diff
@@ -163,3 +163,20 @@ a:hover {
   bottom: 12px;
   right: 14px;
 }
+
+@media (prefers-color-scheme: dark) {
+  .widget-container {
+    --bsky-text-on-theme: #E0E2E6;
+    --bsky-dark: #161C23;
+    color: var(--bsky-text-on-theme);
+  }
+  
+  .card-content-container {
+    background-color: var(--bsky-dark);
+    box-shadow: 1px 1px 8px 1px #000;
+  }
+  
+  .avatar {
+    border: 2px solid var(--bsky-dark);
+  }
+}
```

Comment:
I was thinking of doing this based on a prop like-

```jsx
<bsky-widget 
  // ...
  data-theme="dark"
  data-theme="dim"
  data-theme="light"
></bsky-widget>
```

This way the consumer can decide which card they want to render depending on the theme of their website. If they want to switch it based on prefers-color-scheme, they can do it from their end.

I think that way it will have more expected theming?

Reply:
That seems reasonable, but would be nice to have an "auto" option as well that detects the users preferred theme.

Reply:
oh yes that would be perfect!